try {
    console.log("hello")
} catch (error) {
    console.log(error)
    console.log(error.name)
    console.log(error.message)
}